draft-ietf-mls-protocol.md

```diff
@@ -944,23 +947,32 @@ Given these inputs, the derivation of secrets for an epoch
 proceeds as shown in the following diagram:
 
 ~~~~~
-               init_secret_[n-1] (or 0)
-                     |
-                     V
-update_secret -> HKDF-Extract = epoch_secret
-                     |
-                     +--> Derive-Secret(., "app", GroupState_[n])
-                     |    = application_secret
-                     |
-                     +--> Derive-Secret(., "confirm", GroupState_[n])
-                     |    = confirmation_key
-                     |
-                     V
-               Derive-Secret(., "init", GroupState_[n])
-                     |
-                     V
-               init_secret_[n]
-~~~~~
+                  init_secret_[N-1] (or 0)
+                        |
+                        V
+update_secret_[N-1] --> HKDF-Extract = epoch_secret_[N]
+                        |
+                        +--> Derive-Secret(., "app",
```

Comment:
If you're going to change `Derive-Secret` to `HKDF-Expand-Label`, then you need to chance these instances as well.  Alternatively, you could just define `Derive-Secret` in terms of `HKDF-Expand-Label`, basically as an abbreviation to make this diagram flow better.